Week-9/todoapplication/src/components/Content.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
no use of this component

Reply:
I'm keeping it although it is of no use because... it helps me visualize the component structure of this app well...